developer_docs/how_to_create_a_plugin.asciidoc

```diff
@@ -1678,6 +1678,42 @@ The settings are strongly typed and you have to define it.
 The basic types supported by Foreman are: `:boolean`, `:integer`, `:float`, `:string`, `:text`, `:hash`, `:array`.
 The `:text` type supports markdown and usage of such setting should expect markdown syntax when using it.
 
+==== Validate setting value
```

Comment:
This doesn't read that well. I think something like this is better English
```suggestion
==== Validate a setting's value
```

Or
```suggestion
==== Validate setting values
```